reana_client/cli/files.py

```diff
@@ -318,59 +309,41 @@ def upload_files(ctx, workflow, filenames, access_token):  # noqa: D301
                     response = upload_to_server(workflow, filename, access_token)
                     for file_ in response:
                         if file_.startswith("symlink:"):
-                            click.echo(
-                                click.style(
-                                    "Symlink resolved to {}. Uploaded"
-                                    " hard copy.".format(file_[len("symlink:") :]),
-                                    fg="green",
-                                )
+                            display_message(
+                                "Symlink resolved to {}. "
+                                "Uploaded hard copy.".format(file_[len("symlink:") :])
                             )
                         else:
-                            click.echo(
-                                click.style(
-                                    "File {} was successfully "
-                                    "uploaded.".format(file_),
-                                    fg="green",
-                                )
+                            display_message(
+                                "File {} was successfully uploaded.".format(file_)
```

Comment:
Given what @VMois [commented here](https://github.com/reanahub/reana-client/pull/553#issuecomment-920711731) (`create/upload/run` section): _"This padded message is too much"_, I would rather switch to `click.secho` here.

Although it would inconsistent with the padded errors shown in the exception handling block below, **it is consistent with [your previous comment](https://github.com/reanahub/reana-client/pull/553#discussion_r709977083)** (a.k.a. _when there is a symbolic just switch to `click.secho`_).

Reply:
Changes in https://github.com/reanahub/reana-client/pull/553/commits/841d038bea862dbc40c6888640112d807fc4e4a5